docs/src/app/(docs)/working-with-files/page.mdx

```diff
@@ -18,6 +18,8 @@ with them. This page shows how to work with your uploaded files.
     Do not use the raw file URL from the storage provider, e.g. `https://bucket.s3.region.amazonaws.com/<FILE_KEY>`. We reserve the right to move objects between different storage providers and/or buckets, so this URL is not guaranteed to remain valid.
 </Warning>
 
+# TODO: REWRITE THIS ENTIRE SECTION
```

Comment:
cc @markflorkowski can you look at this section and see what you think we should do. I sort of feel like 90% can just be removed and just list the format? What do you tihnk?

Reply:
Yeah, with the new subdomain change, there is no need for any of this. The only important information is:
- "don't use storage urls"
-  "the correct url format is `https://<appid>.ufs.sh/f/<fileKey | customId>`"
- "heres how you put it in Next.js"